### ISSUE TITLE
Unknown suffixes in internal links & image embedding in HTML export & Pandoc’s fenced-divs syntax

### ISSUE THREAD

**Issue by 273c** [typora/typora-issues#5674]
**★Issue** 
In internal links, suffixes are automatically added and the following display occurs in HTML. This issue occurs with Typora’s Pandoc export & HTML export, but not with Obsidian. It is a Typora-specific issue.

/in markdown/
```
## a

[【sample】](#a)
```

/in html/
```
<h2 id="a"><span>a</span></h2>

<a href="#a-1"><span>【sample】</span></a> 　←This issue
```

**★Request** 
(1)Please add support for image embedding in HTML export (default).
 This can be achieved with “—embed-resources” in Pandoc.

(2)Please also add support for Pandoc’s fenced-div syntax. I love this feature.

 Typora is perfect in every way.

**Comment by 273c** [comment]
sorry changed 
```
<h1 id="a"><span>a</span></h1>

<a href="#a-1"><span>【sample】</span></a> 　←This issue
```

**Comment by abnerlee** [comment]
```
<a href="#a-1"><span>【sample】</span></a> 　←This issue
```

This should be fixed in version 1.6
